effekt/jvm/src/main/scala/effekt/Driver.scala

```diff
@@ -108,6 +108,97 @@ trait Driver extends kiama.util.Compiler[EffektConfig, EffektError] { outer =>
     afterCompilation(source, config)(context)
   }
 
+  def generateCrashReport(e: Throwable, context: Context, config: EffektConfig): Unit = {
+    import java.nio.file.{Files, Paths}
+    import java.util.Properties
+
+    // Capture general information
+    val effektVersion = effekt.util.Version.effektVersion
+    val osInfo = System.getProperty("os.name") +
+      " (" + System.getProperty("os.arch") + ") " +
+      System.getProperty("os.version")
+
+    val jvmInfo = System.getProperty("java.version") +
+      " (" + System.getProperty("java.vendor") + ")"
+
+    val stackTrace = e.getStackTrace.map(line => "  at " + line.toString).mkString("\n")
```

Comment:
What's the difference between this and `.printStackTrace` (to a print writer?)

Reply:
As far as I'm concerned, `.printStackTrace` formats in a JVM Style and prints immediatly. I think the style would be exactly the same, but I want to have the stacktrace in the `markdown`-log file aswell.

Reply:
There's a `PrintWriter` variant of `.printStackTrace`: https://docs.oracle.com/javase/8/docs/api/java/lang/Throwable.html#printStackTrace-java.io.PrintWriter-

Reply:
What would you say is the benefit of this? If I get the `StackTraceover` over a `PrintWriter`, I guess we still have to convert it to a String later for `.md` and `console` logging?
Or do you think a PrintWriter for the whole console report would be beneficial? There again, we would need to get a String copy for the `.md`

Let me know your thoughts:) maybe I'm getting something completly wrong

Reply:
Something like this?:)
```scala
def getStackTrace(e: Throwable): String = {
    val stringWriter = new java.io.StringWriter()
    e.printStackTrace(new java.io.PrintWriter(stringWriter))
    stringWriter.toString
  }
```